// proto/messages.proto
syntax = "proto3";

package cppplumberd;


message CommandHeader {
	uint32 command_type = 1;
}
// Response to commands
message CommandResponse {
  uint32 status_code = 1;
  string error_message = 2;
  uint32 response_type = 3; // 0 means void, if > 0, than it the MessageId.
}

message EventHeader {
	uint64 timestamp = 1;
	uint32 event_type = 2;
}

